src/FreeScribe.client/client.py

```diff
@@ -559,22 +554,19 @@ def send_text_to_api(edited_text):
             app_settings.editable_settings["Model Endpoint"] = app_settings.editable_settings["Model Endpoint"][:-1]
 
         if app_settings.API_STYLE == "OpenAI":
-            response = requests.Response
-            if str(app_settings.SSL_SELFCERT) == "1" and str(app_settings.SSL_ENABLE) == "1":
-                response = requests.post(app_settings.editable_settings["Model Endpoint"]+"/chat/completions", headers=headers, json=payload, verify=False)
-            else:
-                response = requests.post(app_settings.editable_settings["Model Endpoint"]+"/chat/completions", headers=headers, json=payload)
+            verify = True if app_settings.editable_settings["AI Server Self-Signed Certificates"] is True else False
```

Comment:
https://docs.sourcery.ai/Reference/Default-Rules/refactorings/boolean-if-exp-identity/

Fixed in https://github.com/ClinicianFOCUS/FreeScribe/pull/113/commits/c0333ee2d112776b4ed560b091880ac7a63f2663

Inline var immediately returned OK, gives us easy manipulation if we need it

Casts are needed because sometimes the settings.txt force returns a string instead of correct type